clusterctl/clusterdeployer/clusterdeployer.go

```diff
@@ -411,92 +438,98 @@ func pivot(from, to ClusterClient) error {
 			return fmt.Errorf("error moving MachineSet '%v': %v", machineSet.GetName(), err)
 		}
 		glog.Infof("Moved MachineSet %v", machineSet.GetName())
 	}
 
-	machines, err := from.GetMachineObjects()
+	machines, err := from.GetMachineObjectsInNamespace(namespace)
 	if err != nil {
 		return err
 	}
 
 	for _, machine := range machines {
 		// New objects cannot have a specified resource version. Clear it out.
 		machine.SetResourceVersion("")
-		if err = to.CreateMachineObjects([]*clusterv1.Machine{machine}); err != nil {
+		if err = to.CreateMachineObjects([]*clusterv1.Machine{machine}, namespace); err != nil {
 			return fmt.Errorf("error moving Machine '%v': %v", machine.GetName(), err)
 		}
 		glog.Infof("Moved Machine '%s'", machine.GetName())
 	}
 	return nil
 }
 
-func deleteObjects(client ClusterClient) error {
+func deleteObjectsInNamespace(client ClusterClient, namespace string) error {
 	var errors []string
-	glog.Infof("Deleting machine deployments")
-	if err := client.DeleteMachineDeploymentObjects(); err != nil {
+	glog.Infof("Deleting machine deployments in namespace %q", namespace)
+	if err := client.DeleteMachineDeploymentObjectsInNamespace(namespace); err != nil {
 		err = fmt.Errorf("error deleting machine deployments: %v", err)
 		errors = append(errors, err.Error())
 	}
-	glog.Infof("Deleting machine sets")
-	if err := client.DeleteMachineSetObjects(); err != nil {
+	glog.Infof("Deleting machine sets in namespace %q", namespace)
+	if err := client.DeleteMachineSetObjectsInNamespace(namespace); err != nil {
 		err = fmt.Errorf("error deleting machine sets: %v", err)
 		errors = append(errors, err.Error())
 	}
-	glog.Infof("Deleting machines")
-	if err := client.DeleteMachineObjects(); err != nil {
+	glog.Infof("Deleting machines in namespace %q", namespace)
+	if err := client.DeleteMachineObjectsInNamespace(namespace); err != nil {
 		err = fmt.Errorf("error deleting machines: %v", err)
 		errors = append(errors, err.Error())
 	}
-	glog.Infof("Deleting clusters")
-	if err := client.DeleteClusterObjects(); err != nil {
+	glog.Infof("Deleting clusters in namespace %q", namespace)
+	if err := client.DeleteClusterObjectsInNamespace(namespace); err != nil {
 		err = fmt.Errorf("error deleting clusters: %v", err)
 		errors = append(errors, err.Error())
 	}
+	glog.Infof("Deleting namespace %q", namespace)
+	if err := client.DeleteNamespace(namespace); err != nil {
+		err = fmt.Errorf("error deleting namespace: %v", err)
+		errors = append(errors, err.Error())
+	}
 	if len(errors) > 0 {
 		return fmt.Errorf("error(s) encountered deleting objects from bootstrap cluster: [%v]", strings.Join(errors, ", "))
 	}
 	return nil
 }
 
-func getClusterAPIObjects(client ClusterClient) (*clusterv1.Cluster, *clusterv1.Machine, []*clusterv1.Machine, error) {
-	machines, err := client.GetMachineObjects()
+func deleteObjects(client ClusterClient) error {
```

Comment:
Is this method used anywhere still?

Reply:
No. I thought I had removed it. This time really removing it :)